recipes/openssl/3.x.x/conanfile.py

```diff
@@ -85,11 +85,13 @@ class OpenSSLConan(ConanFile):
         "no_whirlpool": [True, False],
         "no_zlib": [True, False],
         "openssldir": [None, "ANY"],
+        "securitylevel":  [None] + list(range(0, 6)),
```

Comment:
```suggestion
        "securitylevel": [0, 1, 2, 3, 4, 5],
```

Use config options to change it:

```
def config_options(self):
    self.options.tls_security_level = 1 Version(self.version) < "3.2" else 2
```

Plus, I would ask to rename the option to `tls_security_level` to reflect its origin.

Reply:
I like the suggested change about the config_options and the specification of valid values. It is clearer. Thanks.
Regarding the rename, I think securitylevel is fine, taking into account the OpenSSL documentation uses more the short term than the whole definition name.

Reply:
>  I think securitylevel is fine, taking into account the OpenSSL documentation uses more the short term than the whole definition name.

Sorry, but I'll insist for `tls_security_level`. The cmake definition from the upstream follows the same name. Other options in this recipe (except openssldir) use underscore to separate option names.

Reply:
Ok. I updated it